### PR TITLE
feat(multi-select): 增加下拉选择器multi-select组件

### PR DESCRIPTION
**Pull request by TC-twwang**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/multi-select/vue.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+export const renderless = (props, { onMounted, reactive, watch }, { emit, nextTick, refs }) => {
+  const api = {}
+  const state = initState({ reactive })
+
+  initApi({ api, props, state, emit, nextTick, refs })
+
+  initWatch({ watch, props, state, refs, nextTick })
+
+  onMounted(() => {
+    api.created({ props, state, refs, nextTick })
```
Comment (shenjunjian) [inline src/multi-select/vue.js:89]: 单参时，可以不传对象

File: src/wheel/index.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+* THE OPEN SOURCE SOFTWARE IN THIS PRODUCT IS DISTRIBUTED IN THE HOPE THAT IT WILL BE USEFUL,
+* BUT WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS FOR
+* A PARTICULAR PURPOSE. SEE THE APPLICABLE LICENSES FOR MORE DETAILS.
+*
+*/
+
+import { cloneDeep } from '@opentiny/vue-renderless/chart-core/deps/utils'
+
+export const created = ({ api }) => () => {
+  api._loadPickerData()
```
Comment (shenjunjian) [inline src/wheel/index.js]: 函数不需要用  _ 打头

File: src/wheel/vue.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+export const renderless = (props, { onMounted, reactive, watch }, { emit, nextTick, refs }, { BScroll }) => {
+  const api = {}
+  const state = initState({ reactive })
+
+  initApi({ api, props, state, emit, nextTick, refs, BScroll })
+
+  initWatch({ watch, api, props, state, nextTick })
+
+  onMounted(() => {
+    api.created({ api })
```
Comment (shenjunjian) [inline src/wheel/vue.js]: 单参不需要传递一个对象

File: src/multi-select/index.js [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+    // 首次点击
+    state.showWheel = true
+    state.headerIndex = value
+    state.headerInfo[value] = getNewHeaderInfo(state.headerInfo, value, true)
+    state.defaultSelectedIndexs = state.defaultSelectedArray[value] ?? api._loadDefault(value)
+  } else if (state.headerIndex !== value) {
+    // 切换
+    state.showWheel = true
+    state.headerInfo[state.headerIndex] = getNewHeaderInfo(state.headerInfo, state.headerIndex, false) // 上一个
+    state.headerIndex = value
```
Comment (zzcr) [inline src/multi-select/index.js:41]: 注释需要和上一行代码间隔一个换行，保持代码的可读性

**Review by kagol — APPROVED** [review]

**Review by kagol — APPROVED** [review]

**Pull request merged**